Notation.Plugin.AzureKeyVault/Certificate/CertificateBundle.cs

```diff
@@ -0,0 +1,40 @@
+using System.Security.Cryptography.X509Certificates;
+
+namespace Notation.Plugin.AzureKeyVault.Certificate
+{
+    /// <summary>
+    /// Helper class to create a certificate bundle from a PEM file.
+    /// </summary>
+    class CertificateBundle
+    {
+        /// <summary>
+        /// Create a certificate bundle from a PEM file.
+        /// </summary>
+        public static X509Certificate2Collection Create(string pemFilePath)
```

Comment:
Will [X509Certificate2Collection.ImportFromPemFile(String)](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2collection.importfrompemfile?view=net-6.0) simply work?

Reply:
The awesome! Thanks Shiwei!